From: Free Ekanayaka <freee@debian.org>
Date: Sat, 11 Feb 2017 15:57:16 +0000
Subject: fix-go-imports

Fix Go imports that still point to the upstream package.
---
 cmd/bolt/main.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmd/bolt/main.go b/cmd/bolt/main.go
index b96e6f7..221012b 100644
--- a/cmd/bolt/main.go
+++ b/cmd/bolt/main.go
@@ -19,7 +19,7 @@ import (
 	"unicode/utf8"
 	"unsafe"
 
-	"github.com/boltdb/bolt"
+	"github.com/NebulousLabs/bolt"
 )
 
 var (
